Makefile

```diff
@@ -39,6 +39,8 @@ DOCKER_BUILD = docker build --no-cache \
 	    PLATFORM=$(PLATFORM) \
 	    BUILD_NUMBER=$(BUILD_NUMBER) \
 	    ENABLE_DHCP_GRAPH_SERVICE=$(ENABLE_DHCP_GRAPH_SERVICE) \
+	    PASSWORD=$(PASSWORD) \
```

Comment:
Ah, gotcha. Pretty sure Guohan's intention was to have them initially set to the defaults in rules/config and only overwritten if explicitly specified on the command line.